packages/connectivity/connectivity_platform_interface/lib/src/utils.dart

```diff
@@ -14,7 +14,7 @@ ConnectivityResult parseConnectivityResult(String state) {
 }
 
 /// Convert a String to a LocationAuthorizationStatus value.
-LocationAuthorizationStatus parseLocationAuthorizationStatus(String result) {
+LocationAuthorizationStatus parseLocationAuthorizationStatus(String? result) {
```

Comment:
looks like `result` could be non-null, right?

Reply:
done